meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-boot.mount

```diff
@@ -7,7 +7,7 @@ Conflicts=umount.target
 [Mount]
 What=/dev/disk/by-label/resin-boot
 Where=/mnt/boot
-Type=vfat
```

Comment:
We can't make this modification this way as this will go into all the other platforms. We will need to have special mounts for UBIFS boards. And maybe each device would be able to select the mounts by using a VARIABLE like: UBIFS_STORAGE = 1 in the resin-board repository.